crates/ruff_python_formatter/src/statement/stmt_class_def.rs

```diff
@@ -117,14 +117,22 @@ impl FormatNodeRule<StmtClassDef> for FormatStmtClassDef {
             }
         }
 
-        write!(
-            f,
-            [
-                text(":"),
-                trailing_comments(trailing_definition_comments),
-                block_indent(&body.format().with_options(SuiteKind::Class))
-            ]
-        )
+        // TODO(tjkuson): determine why this is necessary.
+        if contains_only_an_ellipsis(body, f.context().comments()) {
+            write!(
+                f,
+                [text(":"), clause_body(body, trailing_definition_comments)]
+            )
+        } else {
+            write!(
+                f,
+                [
+                    text(":"),
+                    trailing_comments(trailing_definition_comments),
+                    block_indent(&body.format().with_options(SuiteKind::Class))
+                ]
+            )
+        }
     }
```

Comment:
It may have been because I incorrectly suggested to use `has_comments` and we'll need a way to pass the `SuiteKind::Class` to `clause_body`, e.g. by adding a `with_suite_kind` method to it